Bookish.Web/Controllers/HomeController.cs

```diff
@@ -48,20 +48,24 @@ public IActionResult Copies(string isbn)
             return View(model);
         }
 
-        public IActionResult NewBook()
+        [ActionName("NewBook")]
```

Comment:
You don't need the annotation if the action name is the same as the method name 🙂